Documentation/build_qubes_packages.md

```diff
@@ -0,0 +1,320 @@
+# Qubes OS Packages
```

Comment:
How this documentation can be found?
- there is not mention of it in README
- maybe reference in TrenchBoot Documentation which is published https://trenchboot.org/ should be considered?

TrenchBoot as Qubes OS AEM should gain more visibility on TrenchBoot website. 

Also whenever we have some progress to share it is worthy to try engage community on #trenchboot channel at OSFW slack (and in Dasharo Matrix Space).